src/backend/distributed/citus--6.0-17--6.0-18.sql

```diff
@@ -0,0 +1,22 @@
+/* citus--6.0-17--6.0-18.sql */
+
+SET search_path = 'pg_catalog';
+
+DROP FUNCTION master_add_node(nodename text, nodeport integer);
+
+CREATE FUNCTION master_add_node(nodename text,                                                
+                                nodeport integer,                                             
+                                OUT nodeid integer,                                           
+                                OUT groupid integer,                                          
+                                OUT nodename text,                                            
+                                OUT nodeport integer,                                         
+                                OUT noderack text,                                            
+                                OUT hasmetadata boolean)                                      
+    RETURNS record                                                                            
+    LANGUAGE C STRICT                                                                         
+    AS 'MODULE_PATHNAME', $$master_add_node$$;
+COMMENT ON FUNCTION master_add_node(nodename text,                                            
```

Comment:
(Minor) Could we collapse both arguments onto a single line? I tried that and it looks cleaner to me.